trunk/admin.widget.class.php

```diff
@@ -153,6 +154,10 @@ public function sanitize($input)
             $new_input['from_name'] = trim($input['from_name']);
         }
 
+        if (!empty($input['template'])) {
+            $new_input['template'] = trim($input['template']);
```

Comment:
Actually you also need to use `esc_attr`. Seems we're missing that for `from_email` and `from_name` too.